### ISSUE TITLE
cannot reprodce the results

### ISSUE THREAD

**Issue by Fly-dream12** [tiancity-NJU/da-faster-rcnn-PyTorch#26]
I have downloaded the dataset as you uploaded and trained the baseline model and tranfer model,
however, the model with domain adaption only got AP of 0.2901 while the baseline model can obtain AP of 0.4622, which is quite different from the results in this repo (Our model could arrive mAP=30.71% in target domain which is high than baseline mAP=24.26%). So what may be the reasons?

**Comment by kinredon** [comment]
This should be reasonable, the baseline mAP = 24.26% may the source model test on the target domain.

**Comment by Fly-dream12** [comment]
What is the source model, is it mean the model trained on cityscape train set?

**Comment by Fly-dream12** [comment]
Does it mean run the trainval_net.py and change the line 206 to s_imdb_name to train and test on the target domain?

**Comment by kinredon** [comment]
The source model, as you said, means the model trained on the cityscape train set, and test on foggy cityscape test set.

**Comment by tiancity-NJU** [comment]
You should run da_trainval_net.py. cityscape with true label is source domain and cityscape-foggy with fake label is target-domain.  The final model need to be test on cityscape-foggy.
